docs/semgrep-ci/manual-ci-setup.md

```diff
@@ -0,0 +1,415 @@
+```
+
+### Jenkins
+
+To add Semgrep into your Jenkins pipeline, do the following steps. Your UI may vary depending on your Jenkins installation. These steps use a Classic UI Jenkins interface.
+
+1. Edit or create your `Jenkinsfile` configuration file to add a Semgrep `step` as part of your Pipeline. Refer to the [Jenkins CI code snippet](#jenkins-ci-code-snippet). You can edit your `Jenkinsfile` from Jenkins's interface.
+2. Optional: You can also review [`Jenkinsfile` usage](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/) to help with editing the configuration file.
+3. If you are using Jenkins Classic UI, [save your `Jenkinsfile`](https://www.jenkins.io/doc/book/pipeline/getting-started/#through-the-classic-ui) within the Jenkins interface. Otherwise, commit the configuration file into the root folder within the target repository. The Semgrep job starts automatically upon detecting the `Jenkinsfile` commit in the previous step. 
+4. Optional: Re-run the stage from [Jenkins's interface](https://www.jenkins.io/doc/book/pipeline/running-pipelines/#restarting-from-the-classic-ui).
+6. Customize the job's behavior, such as its rules to scan and files to ignore.
+
+#### Jenkins CI code snippet
+
+This code snippet uses Jenkins declarative syntax.
+
+```javascript
+pipeline {
+  agent any
+    stages {
+      stage('Semgrep-Scan') {
+	      environment { 
+          SEMGREP_RULES = 'p/default'
+        } 
+        steps {
+          sh 'pip3 install semgrep'
+          sh 'semgrep ci'
+      }
+    }
+  }
+}
+```
+
+### BitBucket Pipelines
+
+To add Semgrep into your GitLab CI/CD pipeline, do the following steps.
+
+1. Create or edit a `bitbucket-pipelines.yml` configuration file to add
+   a Semgrep `step` as part of your pipeline. Refer to the [BitBucket Pipelines code snippet](#bitbucket-pipelines-code-snippet).
+2. Optional: You can also review the [Bitbucket guidelines guide](https://support.atlassian.com/bitbucket-cloud/docs/configure-bitbucket-pipelinesyml/) to help with editing the configuration file.
+3. Commit the configuration file into the root folder within the target repository. The Semgrep job starts automatically upon detecting the `bitbucket-pipelines.yml` commit in the previous step.
+4. Optional: Re-run or view the Pipeline by clicking from **your repository > Pipelines**.
+5. Customize the job's behavior, such as its rules to scan and files to ignore.
+
+:::note
+These steps can also be performed through BitBucket's UI wizard. This UI wizard can be accessed through **BitBucket > your repository > Pipelines > Create your first pipeline**.
+:::
+
+#### BitBucket Pipelines code snippet
+
+```yaml
+image: atlassian/default-image:latest
+
+pipelines:
+  default:
+    - parallel:
+      - step:
+          name: 'Run Semgrep scan with current branch'
+          deployment: dev
+          image: returntocorp/semgrep
+          script:
+            - export SEMGREP_RULES="p/default" # TODO get confirmation from Holden
```

Comment:
@holdenmcg Is this the right way to set SEMGREP_RULES in bitbucket?

Reply:
Yes, this is exactly it.